build.sh

```diff
@@ -119,15 +119,17 @@ if [ "X${B_CORE}" == "Xy" ]; then
 		./phoenix-rtos-build/build-core-imx6ull.sh
 	elif [ "${TARGET_SUBFAMILY}" == "armv7-stm32" ]; then
 		./phoenix-rtos-build/build-core-stm32.sh
+	elif [ "${TARGET_SUBFAMILY}" == "ia32-qemu" ]; then
+		./phoenix-rtos-build/build-core-ia32-qemu.sh
```

Comment:
wouldn't it be nicer to just make build-core-$TARGET_SUBFAMILY naming convention for theese scripts instead of adding if case for every new target?

Reply:
Fixed